### DISCUSSION TITLE
Adding CC:Tweaked to Approved Non-CurseForge Mods list?

### DISCUSSION THREAD

**Discussion by w1257** [cc-tweaked/CC-Tweaked#2032] [Q&A] [answered]
Since CC:Tweaked is no longer published on CurseForge and it is becoming outdated on it, Is it possible to get CC:Tweaked added to Approved Non-CurseForge Mods list?

As far as I understand, this will allow CurseForge modpacks to use this mod outside CurseForge and allow the mod to be up to date.

Tell me if I misunderstood or missing something.

update: It was added.

**Comment by SquidDev** [comment, marked as answer]
CC:T has no restrictions on what modpacks it can be included in. As long as the terms of [the license](https://github.com/cc-tweaked/CC-Tweaked/blob/4dc649d5e579e9eeb71a6488d44defd509890285/LICENSES/LicenseRef-CCPL.txt) are followed, the mod may be redistributed anywhere.
